domains/auth/Auth.API/API/Controllers/TermsController.cs

```diff
@@ -48,22 +61,25 @@ public class TermsController : ControllerBase
 
         await userService.UpsertUserAsync(user);
 
-        if (AuthenticationHeaderValue.TryParse(accessor.HttpContext?.Request.Headers.Authorization, out var authentication))
+        if (env.IsDevelopment() is false)
```

Comment:
Let us gate this by checking the optionalness of the uri property of the options.